_includes/landing-page/popular-videos.html

```diff
@@ -8,19 +8,19 @@ <h5 class="title">Popular videos</h5>
     </div>
     <div class="row">
       <div class="col-xs-12 col-md-6 col-lg-12">
-        <a class="video-item" href="https://docker.events.cube365.net/docker/dockercon/content/Videos/hgMFTyX5kYKmTPWZo" target="_blank">
+        <a class="video-item" href="https://youtu.be/iqqDU2crIEQ?t=31" target="_blank">
```

Comment:
minor nit, maybe start this video at `t=30` as well.
Otherwise looks good to me :-)

Reply:
Thanks @StefanScherer. The 'Session reminders' slide is still visible at 30s, therefore starting this at 31s.